e2e/testcases/sync_ordering_test.go

```diff
@@ -487,9 +592,11 @@ func TestDependencyWithReconciliation(t *testing.T) {
 	nt.WaitForRootSyncSyncError(configsync.RootSyncName, applier.ApplierErrorCode,
 		"skipped apply of Pod, bookstore/pod4: dependency apply reconcile timeout: bookstore_pod3__Pod")
 
-	_, err = nomostest.Retry(30*time.Second, func() error {
+	nt.T.Logf("Verify that pod3 is created but not ready and pod4 is not found")
+	_, err := nomostest.Retry(60*time.Second, func() error {
```

Comment:
Maybe we can use `nt.DefaultWaitTimeout` instead? Is there a reason we should use a one-off value here?